db/compaction_job.cc

```diff
@@ -1325,7 +1325,12 @@ Status CompactionJob::OpenCompactionOutputFile(
       sub_compact->compaction->MaxInputFileCreationTime();
   if (output_file_creation_time == 0) {
     int64_t _current_time = 0;
-    db_options_.env->GetCurrentTime(&_current_time);  // ignore error
+    auto status = db_options_.env->GetCurrentTime(&_current_time);
+    // Safe to proceed even if GetCurrentTime fails. So, log and proceed.
+    if (!status.ok()) {
+      ROCKS_LOG_DEBUG(db_options_.info_log,
```

Comment:
worth to be on warning level?

Reply:
mention in the log the time is for getting creation_time in compactions?

Reply:
all good points. Updated the patch with these suggestions.